setupbase.py

```diff
@@ -1,10 +1,18 @@
 # -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Why is this line necessary?

Reply:
I'm not 100% sure to be honest. I had kept it when copying code from: https://github.com/chrissimpkins/fontname.py/blob/master/fontname.py#L20
I did very minimal editing to make it usable as a helper function, but the bulk of the logic is as-is.

I just tried running without it and it seems fine, so I guess it's not really needed.